css-two-column-layout/styles.css

```diff
@@ -0,0 +1,72 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h1 {
+  font-family: 'Karla', sans-serif, serif;
+  text-align: center;
+  justify-content: center;
+  font-size: 2.5rem;
+  margin-left: 200px;
+}
+
+.column-full {
+  height: 80px;
```

Comment:
This should be a different class name, the styling doesn't;t match its name. For a `column` class, the only styling applied to it should be a `width` or `flex-basis`